### PR TITLE
Add moiva

### PR DESCRIPTION
**Pull request by kbakba**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -53,6 +53,7 @@
 - [npm-stats](http://www.npm-stats.com) - Displays metrics about packages.
 - [ghub.io](https://ghub.io) - Redirects to the GitHub repo of an npm package.
 - [npm addict](https://npmaddict.com) - Your daily injection of npm packages.
+- [moiva](https://moiva.io/) - Tool for comparison and discovery packages.
```
Comment (sindresorhus) [inline readme.md]: Typo

Comment (kbakba) [reply readme.md]: updated

File: readme.md [hunk 2]
```diff
@@ -53,6 +53,7 @@
 - [npm-stats](http://www.npm-stats.com) - Displays metrics about packages.
 - [ghub.io](https://ghub.io) - Redirects to the GitHub repo of an npm package.
 - [npm addict](https://npmaddict.com) - Your daily injection of npm packages.
+- [moiva](https://moiva.io/) - Compares and discover packages.
```
Comment (sindresorhus) [inline readme.md]: Another typo

**Comment by kbakba** [comment]
Sorry, my English skills are not so good :( And I don't always understand where is a mistake.

**Pull request merged**